fixtures/schema.js

```diff
@@ -12,7 +12,7 @@ const schema = require('js-schema');
  *
  * @type {string}
  */
-module.exports.VERSION = '3.0.0';
+module.exports.VERSION = '3.0.1';
```

Comment:
Nothing in the schema itself has changed, just more properties are exported for use elsewhere. So the schema version does not need to be increased at all.